### ISSUE TITLE
armeabi support

### ISSUE THREAD

**Issue by ale5000-git** [microg/android_packages_apps_GmsCore#867]
In this change:https://github.com/microg/android_packages_apps_GmsCore/commit/dfb1e7fbb7d6225353cd31b94558a2bce5a2f5a2

What happen if the phone doesn't support armeabi-v7a?
Does the app still works without map?

**Comment by mar-v-in** [comment]
Mapbox seems to be incompatible with armeabi < v7a. I guess this will just crash on older ARMs or maybe can't even be installed

**Comment by ale5000-git** [comment]
@mar-v-in: It would be nice if in this case it just use a dummy map make the rest of the app working (the map may be a secondary part of the app in some cases).

**Comment by voidstarstar** [comment]
When the modular design is complete, armeabi < v7a devices should probably be able to just select a different map back end.

Related to #802.

**Comment by ale5000-git** [comment]
@voidstarstar: Yes, but it must also cover the case of NO map backend installed for the principle of graceful degradation :)

**Comment by mar-v-in** [comment]
Yes, but no maps is still some implementation work to not cause crashes (for example the method yo create a marker on the map must be implemented and return a dummy marker. It's planned but still requires work.
